pkg/interface/src/components/Omnibox.js

```diff
@@ -119,6 +120,8 @@ export class Omnibox extends Component {
       return;
     }
 
+    query = query.toLowerCase();
+
     ['commands', 'subscriptions', 'groups', 'apps'].map((category) => {
```

Comment:
I do think DRYing these out into a class property would reduce future headaches

Reply:
Can you explain?

Reply:
See https://github.com/urbit/urbit/pull/3286/files#r468706373

Reply:
I mean we aren't really repeating ourselves, though; this same `.map` style is in the `search()` behaviour, but it doesn't matter what order we search for results in. So the order of displaying results is in one place, in the `render()`. (It should probably be outside the `render()`, but that's besides the point.

We're only mentioning the order of results once, so directly above instead of in the environment seems preferable.

Reply:
That follows. The other issue that it fixes is if future things get added into the search, but I suppose that would imply a larger rewrite